packages/smithy-client/src/extended-encode-uri.ts

```diff
@@ -0,0 +1,5 @@
+export function extendedEncodeURI(str: string): string {
```

Comment:
Can you add a comment explaining why this is necessary (RFC compliance)?

Reply:
fixed in 7b93183ab